udine1/index.html

```diff
@@ -10,9 +10,146 @@
+					<h2 class="cheading">I nostri contatti</h2>
+					<div class="glyphicon glyphicon-map-marker" style="color:#0090d2;"></div><div class="content"> Via Lorem ipsum - 33100 Lorem (UD) Italy</div><br>
+					<div class="glyphicon glyphicon-earphone" style="color:#0090d2;"></div><div class="content"> T. +39 0000 00000 / F. +39 0000 00000</div><br>
+					<div class="glyphicon glyphicon-envelope" style="color:#0090d2;"></div> E. <div class="content"><a href="mailto:lorem@loremipsum.it">lorem@loremipsum.it</a></div>
+					</section>
+				</div>
+				<div class="col-lg-6 col-md-6 col-xs-12 no-margin">
+					<section class="newsletter">
+					<h2 class="nheading">Newsletter</h2>
+					<input class="newsletter_input" type="text" name="letter" id="letter" size="46" placeholder="Il tuo indirizzo email..."><br>
+					<input class="newsletter_checkbox" type="checkbox" id="accetto"><p> accetto il trattamento dei dati personali</p>
+					<input class="newsletter_button" type="button" name="button" id="button" value="Iscriviti">
+					</section>
+				</div>
+			</div>
+		</div>
+	  
+	<!-- footer
+      ================================================== -->
+	<footer class="margin-top-40">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-3 col-md-3 col-xs-6">
+					<h3 class="fheading">Azienda <span class="dot"></span></h3>
+					<ul class="link-list">
+						<li><a href="#">Lorem</a></li>
+						<li><a href="#">In eo tempore</a></li>
+						<li><a href="#">Quisquam</a></li>
+						<li><a href="#">Debet in eo tempore</a></li>
+						<li><a href="#">Nema</a></li>
+						<li><a href="#">Sicamet</a></li>
+						<li><a href="#">Numquam</a></li>
+					</ul>
+				</div>
+				<div class="col-lg-3 col-md-3 col-xs-6">
+					<h3 class="fheading">Prodotti <span class="dot"></span></h3>
+					<ul class="link-list">
+						<li><a href="#">Lorem</a></li>
+						<li><a href="#">In eo tempore</a></li>
+						<li><a href="#">Quisquam</a></li>
+						<li><a href="#">Debet in eo tempore</a></li>
+						<li><a href="#">Nema</a></li>
+						<li><a href="#">Sicamet</a></li>
+						<li><a href="#">Numquam</a></li>
+						<li><a href="#">Lorem</a></li>
+						<li><a href="#">In eo tempore</a></li>
+						<li><a href="#">Quisquam</a></li>
+						<li><a href="#">Debet in eo tempore</a></li>
+					</ul>
+				</div>
+				<div class="col-lg-3 col-md-3 col-xs-6">
+					<h3 class="fheading">Soluzioni <span class="dot"></span></h3>
+					<ul class="link-list">
+						<li><a href="#">Lorem</a></li>
+						<li><a href="#">In eo tempore</a></li>
+						<li><a href="#">Quisquam</a></li>
+						<li><a href="#">Debet in eo tempore</a></li>
+						<li><a href="#">Nema</a></li>
+						<li><a href="#">Sicamet</a></li>
+						<li><a href="#">Numquam</a></li>
+						<li><a href="#">Lorem</a></li>
+						<li><a href="#">In eo tempore</a></li>
+						<li><a href="#">Quisquam</a></li>
+					</ul>
+				</div>
+				<div class="col-lg-3 col-md-3 col-xs-6">
```

Comment:
`class="col-lg-3 col-md-3 col-xs-6"` is redundant and can be simplified to `class="col-xs-6 col-md-3"`